playbook/app/pb_kits/playbook/pb_multiple_users/multiple_users.rb

```diff
@@ -4,6 +4,7 @@ module Playbook
   module PbMultipleUsers
     class MultipleUsers < Playbook::KitBase
       prop :reverse, type: Playbook::Props::Boolean, default: false
+      prop :size, type: Playbook::Props::String, default: "xs"
```

Comment:
Is this kit (multiple users) already being used in Nitro?
Are there places that implementing this new default will break the layout or intended design?

@derricklannaman 

Can you do a search in github to verify how many places it's currently being used?

Reply:
@jasperfurniss Thanks for the feedback! Currently, it appears to be used only in Flight Control.
Additionally, the code changes made are backwards compatible, as its use wouldn't change anything existing in Flight Control (or elsewhere) but would allow to pass a new prop if desired.